maas-region/charmcraft.yaml

```diff
@@ -126,3 +126,10 @@ parts:
       "*": src/loki/
     prime:
       - src/loki/loki.yml
+
+config:
+  options:
+    tls_mode:
+      default: ""
+      description: Whether to enable TLS termination at HA Proxy ('termination'), or no TLS ('')
+      type: string
```

Comment:
is this the way to provide boolean values in charms? Is there a bool type?

Reply:
According to https://juju.is/docs/sdk/charmcraft-yaml#heading--config there is a `boolean` type

Reply:
This one will not be a boolean since we expect three modes: nothing, termination, passthrough:

- none: no TLS
- termination: the TLS is terminated to HAProxy service. A second service is created for internal communication
- passthrough: the TLS it passing through HAProxy and is terminated at MAAS region

@wyattrees is currently implementing the `passthrough`. Until it is there, this option may look like a boolean. More on the first PR: https://github.com/canonical/maas-charms/pull/194#discussion_r1733265572